conans/test/command/export_pkg_test.py

```diff
@@ -439,3 +440,42 @@ class TestConan(ConanFile):
         cmakeinfo = load(os.path.join(client.current_folder, "conanbuildinfo.cmake"))
         self.assertIn("set(CONAN_LIBS_HELLO1 mycoollib)", cmakeinfo)
         self.assertIn("set(CONAN_LIBS mycoollib ${CONAN_LIBS})", cmakeinfo)
+
+    def export_pkg_json_test(self):
+
+        def _check_json_output():
+            json_path = os.path.join(self.client.current_folder, "output.json")
+            self.assertTrue(os.path.exists(json_path))
+            json_content = load(json_path)
+            output = json.loads(json_content)
+            self.assertEqual(output["error"], False)
+            self.assertEqual(output["installed"][0]["recipe"]["id"],
```

Comment:
This is confusing. If my package has dependencies, it will have several "installed" packages, won't it? The exported one is the first one? Maybe it deserves a section in the json, explicitly about the package that has been exported?

Reply:
Well it might not be very accurate, but package is marked as "built".
I have uploaded a test with dependencies and `export-pkg` does not seem to compute the required packages, only the recipes